wiki/BioSQL.md

```diff
@@ -261,19 +261,20 @@ Running the unit tests
 ----------------------
 
 Because there are so many ways you could have setup your BioSQL
-database, you have to tell the unit test a few bits of information by
-editing the file Tests/setup\_BioSQL.py and filling in the following
+database, you have to tell the unit test a few bits of information. 
+If you installed biopython using pip then the relevant [**Tests**](https://github.com/biopython/biopython/tree/master/Tests) 
+folder would not have been copied. If this is the case one can 
+perform check out (or export) using:
+
+```bash
+svn export https://github.com/biopython/biopython/trunk/Tests
```

Comment:
Please don't recommend using ``svn`` here. If you want to avoid needing ``git``, we can just download a tar-ball or zip file from GitHub using e.g.

```
wget https://github.com/biopython/biopython/archive/biopython-172.zip
unzip biopython-172.zip
```

Or,

```
wget https://github.com/biopython/biopython/archive/biopython-172.tar.gz
tar -zxvf biopython-172.tar.gz
```

Also this lets us recommend getting the tests to match the version of Biopython installed, which should prevent test failures from a version mis-match (although that would be rare with BioSQL as this code is fairly static these days).